metadata_backend/api/middlewares.py

```diff
@@ -50,6 +53,78 @@ async def http_error_handler(req: Request, handler: Callable) -> Response:
     return http_error_handler
 
 
+def jwt_middleware() -> Callable:
+    """Middleware for validating and authenticating JSON web token.
+
+    :param req: A request instance
+    :param handler: A request handler
+    :raises: HTTP Exception with status code 401 or 403
+    :returns: Successful requests unaffected
+    """
+    @middleware
+    async def jwt_authentication(req: Request, handler: Callable) -> Response:
+        if 'Authorization' in req.headers:
+            # Check token exists
+            try:
+                scheme, token = req.headers.get('Authorization').split(' ')
+                LOG.info('Auth token received.')
+            except Exception as err:
```

Comment:
catching every Exception doesn't seem wise, maybe specify what exceptions you're trying catch?